### PR TITLE
Views added to the pet_favorites_items

### PR DESCRIPTION
**Pull request by cbaja**
- Views added to the pet_favorites_items ( Size, Age, Breeds )
- name_font_size added the XML dimension
- com.makeramen:roundedimageview:2.2.1 added for future used

### CODE REVIEW

**Comment by cbaja** [comment]
@scottrichards @smacgregor

File: app/build.gradle [hunk 1]
```diff
@@ -42,6 +42,7 @@ dependencies {
     compile 'com.android.support:recyclerview-v7:23.1.1'
     apt 'org.parceler:parceler:1.0.4'
     compile 'org.parceler:parceler-api:1.0.4'
+    compile 'com.makeramen:roundedimageview:2.2.1'
```
Comment (smacgregor) [inline app/build.gradle:45]: Had fun with this library in a previous homework assignment I liked it too! Are you thinking about using round pet pictures in our favorites list view?

File: app/src/main/res/layout/pet_favorites_items.xml [hunk 2]
```diff
@@ -19,9 +19,12 @@
         android:layout_toRightOf="@id/petFavImage"
         android:layout_alignParentTop="true"
         android:layout_width="wrap_content"
-        android:text="foo"
+        android:text="Pet Name"
```
Comment (smacgregor) [inline app/src/main/res/layout/pet_favorites_items.xml:22]: food for thought, consider adding strings that appear in the UI to our strings.xml file. A good habit to develop early on.

**Comment by smacgregor** [comment]
good stuff. merging.

**Pull request merged**